src/rewards/voters/addresses.rs

```diff
@@ -0,0 +1,46 @@
+use chain_crypto::{Blake2b256, Ed25519, PublicKey, Signature, Verification};
+use futures::TryFutureExt;
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+pub struct Metadata {
+    pub voting_key: String,
+    pub stake_public_key: String,
+    pub reward_address: String,
+    pub nonce: u64,
+}
+
+#[derive(Deserialize)]
+pub struct Transaction {
+    hash: String,
+    tx_id: String,
+    metadata: Metadata,
+    signature: String,
+    block_index: u32,
```

Comment:
The idea is to match the output of the python script, load it with `serde_json` and process it afterwards.